src/main/java/com/microsoft/jenkins/appservice/commands/DockerDeployCommand.java

```diff
@@ -61,7 +61,6 @@ public void execute(final IDockerDeployCommandData context) {
                 disableSMBShareIfNotSet(webApp, update);
 
                 update.withTags(new HashedMap());
```

Comment:
Could u plz also check that whether we can safely remove the `withTags` trick?

Reply:
I removed it and deployed successfully. Looks like it's no longer required.